build/packages/rpm/server/SPECS/victoriametrics-gateway.spec

```diff
@@ -0,0 +1,41 @@
+%undefine _missing_build_ids_terminate_build
+
+%define copying() \
+%if 0%{?fedora} >= 21 || 0%{?rhel} >= 7 \
+%license %{*} \
+%else \
+%doc %{*} \
+%endif
+
+%global repo            VictoriaMetrics
+%global provider        github.com/VictoriaMetrics/%{repo}
+%global commit          v1.82.1
+
+Name:           percona-victoriametrics-gateway
+Version:        1.82.1
+Release:        1%{?dist}
+Summary:        VictoriaMetrics gateway solution
+License:        Apache-2.0
+URL:            https://%{provider}
+Source0:        https://%{provider}/releases/download/%{commit}/vmutils-linux-amd64-%{commit}-enterprise.tar.gz
+
+
+%description
+%{summary}
+
+
+%prep
+%setup -q -c
+
+
+%install
+install -D -p -m 0755 ./vmgateway-prod %{buildroot}%{_sbindir}/vmgateway
+
+
+%files
```

Comment:
We need to provide a license file explicitly here, that `copying` macro won't do the job because it not in the source package.

Reply:
Aka `%license LICENSE`, but we need to pull it from some place.

Reply:
Reached out to Dave to help me figure this out.

Reply:
A good question to ask is: which license should it be?

Reply:
looks like the answer is in line 11 ;)

Reply:
I'm working with Travis to identify the proper license

Reply:
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/LICENSE
Apache License2.0
If we package upstream product we inherit their license in this case

Reply:
It's a bit more complicated. The `vmgateway` component is available under VM Enterprise license. Travis and the team are working with VictoriaMetrics to identify how we can distribute `vmgateway`.
We agreed on a temporary AGPL3 license in the meantime.